.github/workflows/ci_e2e-unstable.yaml

```
@@ -87,3 +87,158 @@ jobs:
+                    -
+                        php: "8.3"
+                        symfony: "^7.3@beta"
+                        mysql: "8.4"
+
+        env:
+            APP_ENV: ${{ matrix.env || 'test_cached' }}
+            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?charset=utf8mb4&serverVersion=${{ matrix.mysql }}"
+
+        steps:
+            -   name: "Checkout (With Branch)"
+                if: "${{ inputs.branch != '' }}"
```

actionlint (1.7.7)

property "branch" is not defined in object type {ignore-failure: bool}

(expression)